assets/src/application/ui/input/InlineEditInput/InlineEditInfinity.tsx

```diff
@@ -9,6 +9,11 @@ const InlineEditInfinity: React.FC<TextProps> = ({ children, onChange, className
 	const [value, setValue] = useState(children);
 	const [isEditing, setIsEditing] = useState(false);
 
+	// if value updated by parent
+	useEffect(() => {
+		setValue(children);
+	}, [children]);
+
```

Comment:
There was a bug that ticket quantity did not reflect the change when changing the corresponding date capacity.

Reply:
hmm, interesting